app/models/pipeline_run.rb

```diff
@@ -775,6 +844,13 @@ def subsample_suffix
 
   delegate :sample_output_s3_path, to: :sample
 
+  def expt_output_s3_path
```

Comment:
If it works and you have time, this would be a good opportunity to dedup and refactor some code.  `alignment_output_s3_path`, `postprocess_output_s3_path` and now `expt_output_s3_path` all contain essentially the same code. So you could make a helper function to which you would pass `sample.sample_expt_s3_path` as an argument